WalletWasabi/Helpers/EnvironmentHelpers.cs

```diff
@@ -249,5 +260,16 @@ public static string GetFullBaseDirectory()
 
 			return fullBaseDirectory;
 		}
+
+		[DllImport("kernel32.dll", CharSet = CharSet.Auto, SetLastError = true)]
+		private static extern EXECUTION_STATE SetThreadExecutionState(EXECUTION_STATE esFlags);
+
+		public static void KeepSystemAwake()
+		{
+			if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+			{
+				SetThreadExecutionState(EXECUTION_STATE.ES_SYSTEM_REQUIRED);
```

Comment:
If I understand it correctly, I believe return value should be at least logged to know whether the call succeeded or not.